kustomize/monitoring/base/monitoring.yaml

```diff
@@ -125,6 +125,93 @@ data:
       - source_labels: [__meta_kubernetes_service_name]
         action: replace
         target_label: kubernetes_name
+    - job_name: kfserving-monitoring
+      honor_timestamps: true
+      scrape_interval: 15s
+      scrape_timeout: 10s
+      metrics_path: /metrics
+      scheme: http
+      follow_redirects: true
+      relabel_configs:
+      - source_labels: [__meta_kubernetes_service_label_networking_internal_knative_dev_serviceType]
```

Comment:
Are these labels on inference service deployments?

Reply:
this configuration is taken from following Prometheus Setup from https://github.com/kserve/kserve/tree/master/docs/samples/metrics-and-monitoring